.github/workflows/pypi_release.yml

```diff
@@ -81,11 +121,11 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           wait-interval: 10
 
-      - name: Upload to pypi
-        run: twine upload dist/*
-        env:
-          TWINE_USERNAME: __token__
-          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+      # - name: Upload to pypi
+      #   run: twine upload dist/*
+      #   env:
+      #     TWINE_USERNAME: __token__
+      #     TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
```

Comment:
does this need to be uncommented? or is t he idea to push to pypi manually ?

Reply:
Thanks for noticing! This was from me testing out the entire build process